jcl/src/java.base/share/classes/java/security/AccessController.java

```diff
@@ -452,7 +458,11 @@ private static void keepalive(Permission... perms) {
  * @see         AccessControlContext
  */
 public static AccessControlContext getContext() {
+/*[IF JAVA_SPEC_VERSION >= 24]*/
+	return new AccessControlContext(new ProtectionDomain[] { new ProtectionDomain(null, null) });
```

Comment:
We can cache a single instance and return it.

Reply:
Created a single instance `ACC_NO_PERM`.